pkg/apis/kubeflow.org/v1/jax_defaults.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func addJAXJobDefaultingFuncs(scheme *runtime.Scheme) error {
```

Comment:
```suggestion
func addJAXDefaultingFuncs(scheme *runtime.Scheme) error {
```

We usually do not add the `Job` prefix in most frameworks, although the XGBoost has the prefix.
Let's keep it simple.